test/version/nmstate.go

```diff
@@ -0,0 +1,35 @@
+package version
+
+import (
+	"fmt"
+	"regexp"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	"github.com/Masterminds/semver"
+
+	"github.com/nmstate/kubernetes-nmstate/test/runner"
+)
+
+var (
+	r = regexp.MustCompile(`nmstate-(.*)-.*`)
```

Comment:
can the `.`  be changed to `\d`? 
make the regexp more accurate and safe?

Reply:
if I do that then I have to also include char '.' and we we discuss about doing proper sematic versioning regexp, the nmstate version is whatever is between "-" chars in case it's not semver we will know with the semver library failing. I don't thing it's worth it to enter complex regexp territory.

Reply:
understand.